generator/tools/templates/react/src/App.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import "./App.css";
+import {
+    initializeApplet,
+    setOperations,
+    Action,
+    Op,
+    Service
+} from "common/rpc.mjs";
+import useEffectHmr from "./hooks/useEffectOnce";
+import { fetchTable, getLoggedInUser } from "./helpers";
+
+class __contract__Service extends Service {
+
+    @Action
+    increment(num: number) {
+        return { num }
+    }
+
+    @Op()
+    async addNum(num: number) {
+        this.increment(num);
+    }
+
+}
+
+const __contract__(lowerCase)Service = new __contract__Service()
+
+
+function App() {
+    const [currentUser, setCurrentUser] = useState("Not logged in");
+    const [error, setError] = useState('')
+
+    const fetchCurrentUser = async () => {
+        setError('')
+        try {
+            const res = await getLoggedInUser();
+            setCurrentUser(res)
+        } catch (e) {
+            setError(`Error loading logged in user ${e}`)
+        }
+    }
+
+
+    useEffectHmr(() => {
```

Comment:
@brandonfancher John and I discussed renaming this function, given it's utility is in not reruning when a hot-reload happens. If you have any more-descriptive names, we're all ears.

Reply:
I realize the HMR causes extra noise in the console. Is it causing more issues than just that?